docs/resource-sharing/overview.md

```diff
@@ -2,10 +2,10 @@
 Compute Resource Sharing Overview
 =================================
 
-OSG uses a resource-overlay ("pilot") model to share resources:
-compute resources are added to a large central resource pool through the use of a bootstrap process, often called a
+OSG uses a resource-overlay ("pilot") model to share resources from your 'local' cluster:
+compute resources are added to a large central resource pool in OSG through the use of a bootstrap process, often called a
 _pilot_ or a _glidein_.
-These pilots, in turn, download and execute user jobs (also known as "payloads") from the resource pool.
+These pilots, in turn, download and execute OSG user jobs (also known as "payloads") from the resource pool to run within the pilots.
```

Comment:
We use OSG in previous sentences a lot so I think the additions of "OSG" here are unnecessary and possibly confusing since OSG is a pretty overloaded term (OSG consortium or OSG VO?)

Reply:
I think it's necessary. For someone who doesn't already understand how things work, it's very confusing. The reason I came into make suggestions was that I had to read over each option several times, myself, and try to insert my prior knowledge - which others will not have - to make sure that I understand.

Reply:
I'm fine with keeping the first one (see suggestion above) but "OSG user jobs" is confusing for folks that know about the OSG VO, which is another large constituency that we have to serve.

Reply:
Can you elaborate on the confusion you're worried about? I'm not sure I follow.

Reply:
OSG user jobs could be understood as "OSG VO user jobs", which makes the meaning of the sentence confusing for other VO in the OSG consortium.

Reply:
"user jobs from OSG"?

Reply:
https://github.com/opensciencegrid/docs/pull/780/files#r577088576 is the happy compromise that we agreed upon